CHANGELOG.md

```diff
@@ -5,7 +5,14 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
-## 0.6.0 2024-04-29
+## 0.7.0 2024-04-04
```

Comment:
Whoops!  I just realized that the api spec changes were made separate from the code changes -- much like tests, the API spec / documentation should be directly connected to the code changes they reference (think of a commit is a coherent block of related changes -- once a PR is merged in, the PR frame is lost and each commit stands on its own in the history!)